scripts/exregional_run_analysis.sh

```diff
@@ -367,6 +367,9 @@ if [ ${regional_ensemble_option:-1} -eq 5 ]  && [ ${BKTYPE} != 1  ]; then
   if [ ${l_both_fv3sar_gfs_ens} = ".true." ]; then
     nummem_gfs=$(more filelist03 | wc -l)
     nummem_gfs=$((nummem_gfs - 3 ))
+  else
+    weight_ens_gfs=1.0
+    weight_ens_fv3sar=1.0
```

Comment:
Should weight_ens_fv3sar be 1.0 and weight_ens_gfs be 0?

Reply:
The options "weight_ens_gfs" and "weight_ens_fv3sar" are effective in GSI only in the case of "l_both_fv3sar_gfs_ens=.true." Both are forced to be 1 in GSI in the case of "l_both_fv3sar_gfs_ens=.false.", so the result is not changed even if any values are set here. (I added this part just in case.)

Reply:
I removed this part according to the discussion of the comment below.

Reply:
Thanks!